index.html

```diff
@@ -1,22 +1,237 @@
+			<div id="thumbnail-container">
+				<ul>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=O_LVl8w4-vw" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/O_LVl8w4-vw/mqdefault_6s.webp?du=3000&sqp=CPTpq4sG&rs=AOn4CLDGr1zzvlqIiI6H7YJ-o3tMYUtmSA"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">The Future of This Channel</p>
+								<p class="thumbnail-description">
+									Rick Beato
+									<br>
+									511,370 views
+									<br>
+									A month ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=NaqySto4bKI" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/NaqySto4bKI/mqdefault_6s.webp?du=3000&sqp=CMqXrIsG&rs=AOn4CLArcSYGrOHfxlGwL8mn03xl_FKf_A"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">How I Sold 1,000,000 Records</p>
+								<p class="thumbnail-description">
+									Rick Beato
+									<br>
+									369,725 views
+									<br>
+									two months ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=k2vqJ78VA4g" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/k2vqJ78VA4g/mqdefault_6s.webp?du=3000&sqp=COiOrIsG&rs=AOn4CLBSyTdE6kqQ_fssYjfPPcvNj4oy6g"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">The Ron Carter Interview</p>
+								<p class="thumbnail-description">
+									Rick Beato
+									<br>
+									231,778 views
+									<br>
+									two months ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=mgkvU5eCaYo" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/mgkvU5eCaYo/mqdefault_6s.webp?du=3000&sqp=COSOrIsG&rs=AOn4CLB3spOKe_IrLJct4TjOalEdWSpUSw"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">Brian May Interview - Queen's Songs, Stories and Guitar Style
+								</p>
+								<p class="thumbnail-description">
+									Rick Beato 2
+									<br>
+									258,474 views
+									<br>
+									three weeks ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=qFVpSjRUD2E" target="_blank">
+								<img id="img"
```

Comment:
You cannot have more than one element with the same id in an HTML document. Please change it to class